src/types/profileManager.ts

```diff
@@ -85,7 +85,7 @@ export function readSSH(mnemonics: string): Promise<string> {
 }
 
 export function storeSSH(mnemonics: string, ssh: string): Promise<boolean> {
-  const metadata = JSON.stringify({ sshkey: ssh });
+  const metadata = JSON.stringify({ sshkey: ssh.trim() });
```

Comment:
when trying to add white space at the end, it tries to store the new value which already is the same one on the kvstore